iele-gas.md

```diff
@@ -374,17 +374,21 @@ which is maintained by the next rules.
          <currentMemory> CURR => CURR +Int maxInt(0, NEWSIZE -Int bytesInWords((lengthBytes(LM)))) </currentMemory>
          <peakMemory> PEAK => maxInt(PEAK, CURR +Int maxInt(0, NEWSIZE -Int bytesInWords((lengthBytes(LM))))) </peakMemory>
 
-    rule <k> #memoryExpand(INDEX, _) ... </k>
-         <localMem> LM (.Map => INDEX |-> .Bytes) </localMem>
+    rule <k> #memoryExpand(INDEX, NEWSIZE) => #deductMemory(PEAK) ... </k>
+         <localMem> LM </localMem>
+         <currentMemory> CURR => CURR +Int maxInt(0, NEWSIZE -Int bytesInWords(0)) </currentMemory>
```

Comment:
What is the maxInt for? Isn't `bytesInWords(0)` always `0`? Is memoryExpand ever used with NEWSIZE negative?